Documentation/installation/k8s-install-broadcom-vmware-esxi-nsx.rst

```diff
@@ -0,0 +1,99 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k8s_install_broadcom_vmware_esxi_nsx:
+
+******************************************
+Installation on Broadcom VMware ESXI / NSX
+******************************************
+
+Cilium can be installed on VMware ESXI with or without NSX by using official image
+
+Deploying Cilium on Broadcom VMware vSphere ESXi with or without NSX(-T)
+========================================================================
+
+Cilium can be deployed on VMware vSphere ESXi, with or without NSX(-T). However, there are known issues when using VXLAN as the encapsulation mode.
+
+Known Issue: Pod Communication Failure Across Hosts
+===================================================
```

Comment:
Rather that framing this install guide as "here are all of the issues" and then following with the mitigation steps, I would suggest following the style for existing guides (such as `Documentation/installation/kind.rst`) and first build up the expected correct way to install first.

This is well written for a troubleshooting article, but as a first step for a user looking to install on this platform I think that the above format would work better to communicate positively what the user should do to have a good experience.